### ISSUE TITLE
training workshop COST Mobilise, Spring 2023

### ISSUE THREAD

**Issue by myrmoteras** [plazi/arcadia-project#206]
Dear @jugiora @JonasBlanco @flsimoes 

This is in regards of the originally planned face to face training course in Brussels after the sprint in Geneva in January. 
What is planned is a two day online training course to liberated data, with a special emphasis on the matching service.
I would suggest that we start from the linking material to specimen and then explain how additional material citation could be generated using GGI. 
How much in depth we want to train people I am not sure, needs to be discussed. Ideally we could train them to use a GGI version that would only allow to annotate material citations I articles that we have already processed.

There is not a lot of time left. but I hope you have by now enough material so we can get this done?!

How do you want to proceed?

Shall we recommend to run the workshop in the week of Feb 14-17 or Feb 21-24?

thanks for your input

donat

BTW: shall we create a training project or repo to keep all these discussions etc in one place?
 --------------------------

From: MARTELLOS STEFANO <martelst@units.it> 
Sent: Wednesday, December 21, 2022 5:41 PM
To: Donat Agosti <agosti@amnh.org>
Cc: Vania Leonardo Ferreira <vania.leonardoferreira@naturalis.nl>; Dimitris Koureas <dimitris.koureas@naturalis.nl>; Ana CASINO <acasino@naturalsciences.be>
Subject: Re: COST mobilise training

EXTERNAL SENDER

Dear Donat, plase, can u provide me a feedback? Lately it seems that my emails are failing to reach the recipient… 

Ciao
Stefano

Il giorno 20 dic 2022, alle ore 13:01, MARTELLOS STEFANO <martelst@units.it> ha scritto:

Dear Donat, 
It would be better to have it or slightly before or immediately after the meeting in Bruxelles, in order to avoid an overload of activities. 
It could be done during the week 13-16 February. I think it could be the better choice.
As far as the announcement is concerne, we could publish and disseminate a call in early January, late December. 
Bests
Stefano

Il giorno 20 dic 2022, alle ore 10:42, Donat Agosti <agosti@amnh.org> ha scritto:

Dear Vânia and Dimitris
 
Thanks for the document which I haven’t seen. I look at the 700EUR in the framework of this program as symbolic for the value of the efforts to make biodiversity literature and natural history libraries content accessible in our community.
 
We will provide the online training listed in the document. Who is and how will it be announced? Is it supposed to be during the final meeting in Brussels?
 
Cheers and Happy New Year
 
Donat
 
From: Vania Leonardo Ferreira <vania.leonardoferreira@naturalis.nl> 
Sent: Tuesday, December 20, 2022 10:08 AM
To: MARTELLOS STEFANO <martelst@units.it>
Cc: Dimitris Koureas <dimitris.koureas@naturalis.nl>; Donat Agosti <agosti@amnh.org>; Ana CASINO <acasino@naturalsciences.be>
Subject: Re: COST mobilise training
 
EXTERNAL SENDER
 
Good morning, 
 
I am sending the WBP.
Should you have any questions, kindly let me know.
 
 
Best regards,
Vânia
 
 
On Tue, 20 Dec 2022 at 09:29, MARTELLOS STEFANO <martelst@units.it> wrote:
My bad guys. I sent an email a few days ago to Donat… Was it lost? I was wondering why I did not receive any feedback, actually. 
Sorry, I should have been more careful in this.
Anyway, I do not have a copy of the final budget plan, can Vania please share it to me as well?
 
Bests
Stefano
 

Il giorno 20 dic 2022, alle ore 09:01, Dimitris Koureas <dimitris.koureas@naturalis.nl> ha scritto:
 
I thought Stefano would communicate the updates, as he's in the core group.
 
@Vania Leonardo Ferreira can you please share the WBP with Donat?

Kind regards,
Dimitris 
 
--
Dr Dimitris Koureas
Managing Director for Digital & Chief Information Officer        
Naturalis Biodiversity Center
 
Executive Director, DiSSCo Research Infrastructure
Director, Biodiversity Genomics Europe | Chair, Mobilise COST Action
co-Chair, FAIR Digital Objects Forum
 
0000-0002-4842-6487 | linkedin | @DimitrisKoureas | +31 (0) 71 751 9258
 
 
On Tue, 20 Dec 2022 at 08:59, Donat Agosti <agosti@amnh.org> wrote:
Hi Dimitris
Do you have a copy of the work and budget plan, or a link to it? I don’t have one.
Cheers
Donat
 
 
From: Dimitris Koureas <dimitris.koureas@naturalis.nl> 
Sent: Tuesday, December 20, 2022 8:58 AM
To: Donat Agosti <agosti@amnh.org>
Cc: Ana CASINO <acasino@naturalsciences.be>; Stefano Martellos <martelst@units.it>; Vania Leonardo Ferreira <vania.leonardoferreira@naturalis.nl>
Subject: Re: COST mobilise training
 
EXTERNAL SENDER
 
Dear Donat, 
 
The Work and Budget plan for the final months of MOBILISE has been approved. This was discussed extensively in the core group of MOBILISE. The TS is included as online training, as we had agreed. 
 
 

Kind regards,
Dimitris 
 
--
Dr Dimitris Koureas
Managing Director for Digital & Chief Information Officer        
Naturalis Biodiversity Center
 
Executive Director, DiSSCo Research Infrastructure
Director, Biodiversity Genomics Europe | Chair, Mobilise COST Action
co-Chair, FAIR Digital Objects Forum
 
0000-0002-4842-6487 | linkedin | @DimitrisKoureas | +31 (0) 71 751 9258
 
 
On Tue, 20 Dec 2022 at 08:55, Donat Agosti <agosti@amnh.org> wrote:
Dear Dimitris
 
Since we haven’t had any feedback from the COST Mobilise team regarding the organization of a training workshop for liberating data from publications, we assume that it is canceled.
 
Merry Christmas and Happy New Year
 
Donat
 
 
Donat Agosti
President
Plazi
ORCID 0000-0001-9286-1200

**Comment by JonasBlanco** [comment]
Hi, Donat!

We are now developing the learning materials, but in mid-February we will certainly have something to present.
At the moment, we are focusing on the presentation regarding the Matching Service, but we also have some material for a basic course about our individual extraction procedures too. We can easily add this matter to the material designed for the Matching Service.
I agree about training people using a GGI version that would only allow annotations over already processed papers, it seems the right thing to do seen that we only have two days for the training.
Since there is not a lot of time left and we have several scheduled meetings regarding the learning platforms that we will use and the development of the learning material itself, I would suggest we try to run the workshop in the week of Feb 21-24. Could it be?

Cheers,

**Comment by myrmoteras** [comment]
I can propose the date and see what they suggest. Probably it has to be in February, beacause the project extension will end at the end of February. 
If you two feel comfortable with the date, then I will propose it.

**Comment by flsimoes** [comment]
Hi Donat,

In this issue, https://github.com/plazi/eBioDiv/issues/144, you
mention February 8-10. If I'm not mistaken that's the actual date of the
COST Mobilise event in Brussels, while "our" bit is the one we are
discussing here, right?

Just want to clear it as we got a bit confused.

Regarding the dates itself, I believe we should suggest Feb 21-24 as it
allows the team a bit more time to prepare, considering the tight schedule
until then.

Cheers
Felipe

**Comment by JonasBlanco** [comment]
Hi @myrmoteras 

![image](https://user-images.githubusercontent.com/92064978/212744106-aae600b3-c6aa-4c0e-83f9-f2a9e8eaf473.png)

I think that we need to define how we will proceed about this course now, so that we can properly reply them.
As Ana Casino mentioned, the idea for the COST Mobilise is to focus on "digitization of small collections". If we want to teach them how to extract materials citations using GGI, we should: 
(1) define to which extent they will have access to Plazi server and which version of GGI we will make available to them (the one that would only allow to annotate material citations in articles that we have already processed?);
(2) propose a separate activity outside Mobilise. If that is the case and/or this was already proposed, we should make it clearer (at least Ana and Marko did not understand it) and try to set the meeting with GBIF as soon as possible.

**Comment by myrmoteras** [comment]
@JonasBlanco let's forget for the moment the small collections in COST Mobilise. This is not a training we are involved - this is and the one on person names are the ones they funded to invite people to face to face. whilst our is independent of this. What we proposed is among others the linking of specimen and material citation as core of the training, but I think we need also to explain TB, basics of GGI.

For the limited editions of GGI we need to talk to Guido - if you think this is important than rather sooner. This is similar to what we need to provide to anestore.

I sent an email to Ana and Katerina to find out - Marko is busy this week, and I am not sure how helpful he is - he is really a PR person for CETAF and BiCIKL.

**Comment by flsimoes** [comment]
I don't think that a limited edition of GGI would be made in time for this particular workshop, at least not without making Guido stop completely what he is doing + the sprint. So, my suggestion is that we work without that possibility in mind, but again... I may be wrong to assume that...

**Comment by JonasBlanco** [comment]
We think that maybe there is no point in teaching people about the basics of GGI if they cannot use it during the workshop and curate their own data by themselves.

So, if Guido don't have enough time to develop this version of GGI, what if we focus our workshop mostly in the linking of specimen to material citation also covering something about TB that they could easily access and explore?

But, if you think that it would be better to teach the basics of GGI, we will include this in the workshop, of course.

**Comment by gsautter** [comment]
Well, it's not all tat complicated to make a slimmed-down version ... it's basically giving the whole setup a name, selecting the desired gizmos from a list, writing a short introduction for the README, and hitting the "Export" button ... what I need there is mainly a rough idea of what exactly versions you guys have in mind.
